common/dmfsnmp.c

```diff
@@ -148,14 +148,17 @@ print_mib2nut_memory_struct(mib2nut_info_t *self)
 /*END DEBUGGING*/
 
 int load_neon_lib(void){
+	char *neon_libname = get_libname("libneon.so");
 
 	if( lt_dlinit() != 0 ) {
 		fprintf(stderr, "Error initializing lt_init\n");
 		upsdebugx(1, "Error initializing lt_init\n");
 		return 0;
```

Comment:
Well... I checked, I couldnt content the curiosity, is not defined err as 0, I fixed, next PR